src/node_messaging.cc

```diff
@@ -609,11 +609,20 @@ void MessagePort::OnMessage() {
   HandleScope handle_scope(env()->isolate());
   Local<Context> context = object(env()->isolate())->CreationContext();
 
+  int count = 0;
+
   // data_ can only ever be modified by the owner thread, so no need to lock.
   // However, the message port may be transferred while it is processing
   // messages, so we need to check that this handle still owns its `data_` field
   // on every iteration.
   while (data_) {
+    if (count++ == 100) {
```

Comment:
Is this an empirical value? Can we use a constant for it?

Reply:
> Is this an empirical value?

It’s a value that I guess is large enough to have no practical performance impact, that’s all.

> Can we use a constant for it?

You mean, a named constant? Sure, although I think the usage here is pretty explicit about what it stands for.